### ISSUE TITLE
Another PSMove centering issue

### ISSUE THREAD

**Issue by RamaRamesses** [TrueOpenVR/TrueOpenVR-Drivers#8]
Anyway.. I think theres another problem
Yes, both controllers will head forward when you press the centering button,
But, after some time the axes would become incorrect and I just realize my roll axis becomes yaw.

After I opened the psmoveconfigtool I notice the psmove controller is still drifted, while in steamvr it is correctly headed towards forward which makes the orientation confused and become uncorrect.

**Comment by r57zone** [comment]
Maybe calibration of PSMoveService will help? It seems there is a possibility of calibration with a magnetometer.

**Comment by RamaRamesses** [comment]
Yup, I had to recalibrate again in psmoveconfigtool, since the tovr centering button doesnt reset the actual angle of the psmoveservice, but it just resets the angle that is displayed in steamvr.

**Comment by r57zone** [comment]
Try this,  reset on PS button on both controllers
[PSMoveService64.zip](https://github.com/TrueOpenVR/TrueOpenVR-Drivers/files/4535148/PSMoveService64.zip)
Test ver 0.5

**Comment by RamaRamesses** [comment]
> Try this, reset on PS button on both controllers
> [PSMoveService64.zip](https://github.com/TrueOpenVR/TrueOpenVR-Drivers/files/4535148/PSMoveService64.zip)
> Test ver 0.5

This version makes the driver crash.

**Comment by r57zone** [comment]
Test ver 0.51
[PSMoveService64.zip](https://github.com/TrueOpenVR/TrueOpenVR-Drivers/files/4535243/PSMoveService64.zip)

**Comment by RamaRamesses** [comment]
> Test ver 0.51
> [PSMoveService64.zip](https://github.com/TrueOpenVR/TrueOpenVR-Drivers/files/4535243/PSMoveService64.zip)

This one can get to SteamVR, but suddenly crashed my bluetooth connection that I had to re pair all my move controllers to my pc.

**Comment by r57zone** [comment]
It works, but they disconnected? Could you try again after rebooting the PC to confirm this behavior?

**Comment by RamaRamesses** [comment]
Yup, I have tried restarting my pc, it crashes my controller again. It was fine with the older driver.
